### PR TITLE
Create a new chapter on shared ownership

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

This chapter was originally discussed and written in an online post by Malvika Sharan, Warrick Ball, Ben Krikler, and Andrew Brown as a part of the [Collaboration Workshop 2021](https://www.software.ac.uk/cw21) speed blog series hosted by [Software Sustainability Institute](https://www.software.ac.uk).

### List of changes proposed in this PR (pull-request)

* Create main chapters and subchapters
* add images
* Add contributors

### What should a reviewer concentrate their feedback on?

- [x] Topic fits the guide
- [x] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [X] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: 
  - Warrick Ball, Ben Krikler, and Andrew Brown

### CODE REVIEW

**Comment by malvikasharan** [comment]
@all-contributors please add @benkrikler for idea and content

**Comment by malvikasharan** [comment]
@all-contributors please add @warrickball for idea and content

**Comment by malvikasharan** [comment]
@all-contributors please add @abrown41 for idea and content

File: book/website/_bibliography/references.bib [hunk 1]
```diff
@@ -621,6 +621,14 @@ @article{McKiernan2016Open
...
 }
 
+@misc{ThalerSunstein2009Nudge,
+	title = {{Nudge by Richard H. Thaler, Cass R. Sunstein: 9780143115267 {$\vert$} PenguinRandomHouse.com: Books}},
+	journal = {PenguinRandomhouse},
+	year = {2009},
+	month = {February},
+	note = {[Online; accessed 5. May 2021]},
+	url = {https://www.penguinrandomhouse.com/books/304634/nudge-by-richard-h-thaler-and-cass-r-sunstein}
+
```
Comment (malvikasharan) [inline book/website/_bibliography/references.bib:631]:
```suggestion
}
```

File: book/website/collaboration/shared-ownership/shared-ownership-challenges.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+(cl-shared-ownership-challenges)=
+# Challenges with Applying Ownership Models Retroactively
+
+While these project ownership models represent the ideals of open source, there are issues that hinder their universal adoption.
+Specifically, it is difficult to apply shared ownership models retroactively for a few reasons described below.
+
+## Strong Ownership
+
+A “strong-ownership” model is When a single contributor or small group of developers feel a strong sense of ownership over the project.
```
Comment (pgadige) [inline book/website/collaboration/shared-ownership/shared-ownership-challenges.md:9]: change uppercase letter W to lowercase letter w in "When" in A "strong-ownership" model is When a single contributor ...

File: book/website/collaboration/shared-ownership/shared-ownership-challenges.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Research software is developed by individuals within research groups, which operate within an organisation or a regulated research environment.
+For instance, nationally-funded research operates under national laws that may not always comply with the legal requirements in other countries.
+Organisations hosting these research themselves are constrained by national laws and international interests.
+The very structure of these institutions depends on ownership.
+For example, we have found that arranging collaboration agreements for joint funding is often held up by the need to define ownership of the intellectual property, even if the software is open source.
+This may be part of the larger legal framework: specifying that software is owned by a project or community rather than a person may be ideal for encouraging participation, but it could also be open to abuse if the legal ownership of the software can be disputed or, worse, appropriated by bad actors.
```
Comment (pgadige) [inline book/website/collaboration/shared-ownership/shared-ownership-challenges.md:22]: I wonder if the comma should appear before or and after disputed.
"...legal ownership of the software can be disputed, or worse, appropriated by bad actors."

File: book/website/collaboration/shared-ownership/shared-ownership-defaults.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+(cl-shared-ownership-defaults)=
+# Nudging for a Better Default
+
+Research has shown that humans making decisions have a strong tendency to go along with the status quo or default option (See the [Nudge Theory](https://www.imperial.ac.uk/nudgeomics/about/what-is-nudge-theory/) by {cite}`ThalerSunstein2009Nudge`).
+In the absence of an actively defined default, the prevailing cultural or institutional norm becomes the de-facto option of choice.
+And hence, the default options for ongoing and new projects should nudge strongly towards adopting a shared ownership model.
+
+The minimum default for an open source project should include the following documents:
+i. defining the open source license,
```
Comment (pgadige) [inline book/website/collaboration/shared-ownership/shared-ownership-defaults.md:9]: The ordered list items are rendered on the same line. Intuitively, the items should be rendered on separate lines.

**Review by pgadige — APPROVED** [review]
Save a few minor modifications, the overall content+structure is approved for merging.

File: book/website/collaboration/shared-ownership/shared-ownership-defaults.md (hunk 4 above)
Comment (malvikasharan) [reply book/website/collaboration/shared-ownership/shared-ownership-defaults.md]: Fixed! Thanks Pooja.

**Comment by malvikasharan** [comment]
Thanks, @pgadige for the review. 
@benkrikler @warrickball or @abrown41, do you have any final comments or would you like to review this before we merge it?

**Comment by malvikasharan** [comment]
@all-contributors please add @pgadige for review

**Pull request merged**